synapse/handlers/profile.py

```diff
@@ -51,6 +52,10 @@ def __init__(self, hs):
                 self._start_update_remote_profile_cache, self.PROFILE_UPDATE_MS,
```

Comment:
it might be a good time to shift this (and the things it calls) to a MasterProfileHandler